UPDATING.md

```diff
@@ -234,6 +234,23 @@ Smart sensors, an "early access" feature added in Airflow 2, are now deprecated
 
 See [Migrating to Deferrable Operators](https://airflow.apache.org/docs/apache-airflow/2.2.4/concepts/smart-sensors.html#migrating-to-deferrable-operators) for details on how to migrate.
 
+### Database configuration moved to new section
+
+The following configurations have been moved from `[core]` to the new `[database]` section.
+
```

Comment:
Information that they can still be accessed via "core" section but will raise a deprecation warning would be useful to add here.

Reply:
This is NIT. And it is good without it - feel free ot add it if you want though.